conf/presto/catalog/pulsar.properties

```diff
@@ -20,7 +20,7 @@
 # name of the connector to be displayed in the catalog
 connector.name=pulsar
 # the url of Pulsar broker service
-pulsar.broker-service-url=http://localhost:8080
+pulsar.web-service-url=http://localhost:8080
```

Comment:
@eaba I don't think we should rename this setting. We should introduce a new setting and deprecate the old setting.

Reply:
`pulsar.web-service-url` as a new setting?